core/v2/assignment.go

```diff
@@ -2,72 +2,191 @@ package v2
 
 import (
 	"fmt"
-	"math"
-	"math/big"
+	"sort"
 
-	"github.com/Layr-Labs/eigenda/common"
 	"github.com/Layr-Labs/eigenda/core"
 )
 
-// getMaxStakePercentage calculates the maximum stake percentage across specified quorums for a specific operator.
-// The function assumes that it will be passed an operator state that contains the quorums for which the operator is a member.
-// Therefore, it does not return an error if a quorum is not found in the state.
-func getMaxStakePercentage(state *core.OperatorState, quorums []core.QuorumID, id core.OperatorID) (*big.Float, error) {
-	maxStakePercentage := new(big.Float)
-	found := false
+func getOrderedOperators(state *core.OperatorState, quorum core.QuorumID) ([]core.OperatorID, map[core.OperatorID]*core.OperatorInfo, error) {
```

Comment:
nit, a lot of the functions declarations in this file are on very long lines. Would it be possible to wrap these long lines?